backend/tests/v5/unit/processors/teamspaces/projects/projects.test.js

```diff
@@ -157,10 +164,93 @@ const testGetAllTemplates = () => {
 	});
 };
 
+const testGetImage = () => {
+	describe('Get image', () => {
+		test('should get image stream', async () => {
```

Comment:
from your implementation we're not gettig image stream, we're getting the blob in this entirety?